src/lib/molecules/footer.svelte

```diff
@@ -1,57 +1,71 @@
 <script>
-    import { LinkButton, LogoBlue } from '$lib/index.js'
+	import { LinkButton, LogoBlue } from '$lib/index.js';
+
+	function navigateTo(url) {
+        window.location.href = url;
+    }
 </script>
+
 <footer>
-    <article>
-        <nav>
-            <a href="/">Home</a>
-            <a href="/leeslijst">Leeslijst</a>
-            <a href="/uitleningen">Uitleningen</a>
-            <a href="/reserveringen">Reserveringen</a>
-            <a href="/betalingen">Betalingen</a>
-          </nav>
-    </article>
-    <article>
-        <a href="/">
-            <img src="{LogoBlue}" alt="oba logo" width="200">
-        </a>
-    </article>
+	<article>
+		<nav>
+			<button on:click={() => navigateTo('/')}>Home</button>
+            <button on:click={() => navigateTo('/leeslijst')}>Leeslijst</button>
+            <button on:click={() => navigateTo('/uitleningen')}>Uitleningen</button>
+            <button on:click={() => navigateTo('/reserveringen')}>Reserveringen</button>
+            <button on:click={() => navigateTo('/betalingen')}>Betalingen</button>
+		</nav>
+	</article>
+	<article>
+		<button on:click={() => navigateTo('/')}>
+            <img src={LogoBlue} alt="oba logo" width="200" />
+        </button>
```

Comment:
Geen buttons gebruiken voor navigatie naar interne of externe url's. Dit met buttons en clients side javascript oplossen is bad practise; niet toegankelijk, niet robuust (want je maakt geen gebruik van web standaarden) en je maakt het onnodig complex.

Maak een breakdown van het nav element met semantische elementen!

Reply:
Als je vanuit de error pagina op de navigatie in de footer drukt werkt de CSS van die pagina niet. Ik heb dit gefixt door dit te doen. Weet jij misschien hoe ik forceer dat de CSS refreshed ofzo als er op een link gedrukt wordt?

Reply:
Hier ben je inmiddels uit gekomen toch?

Reply:
ja ik heb de fix al gecommit